frontend/coprs_frontend/coprs/logic/coprs_logic.py

```diff
@@ -1262,7 +1262,9 @@ def filter_outdated(cls, query):
                      .filter(models.CoprChroot.deleted.isnot(True))
 
                      # Filter only inactive (i.e. EOL) chroots
-                     .filter(not_(models.MockChroot.is_active)))
+                     .filter(or_(
+                         not_(models.MockChroot.is_active),
+                         models.MockChroot.rolling.is_(True))))
```

Comment:
This is hard to validate :-/ sorry.  The `is_active` check exists to not "notify" users about disabled chroots, only EOLed.

Reply:
No, I don't get why we limit to only non-active chroots, but ... I had to relax this with `or_` to a) notify user so b) we can later remove.  We don't b) without a).